features/official_copy_legal_text.feature

```diff
@@ -0,0 +1,41 @@
+@official_copy
+Feature: Legal requirement updates made to Official Copy of the Register
```

Comment:
As it's just one use case (viewing the PDF), it would be best for it to be one scenario, with the existing assertions in it.